# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/stf/pegasus/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- pegasus-2.3.2/src/Pegasus/Common/Platform_LINUX_IX86_GNU.h	2003-10-22 14:26:03.000000000 +0000
+++ pegasus-2.3.2-patched/src/Pegasus/Common/Platform_LINUX_IX86_GNU.h	2004-06-20 01:17:37.000000000 +0000
@@ -59,7 +59,7 @@
 
 #define PEGASUS_HAVE_FOR_SCOPE
 
-// #define PEGASUS_HAVE_TEMPLATE_SPECIALIZATION
+#define PEGASUS_HAVE_TEMPLATE_SPECIALIZATION
 
 #ifndef _GNU_SOURCE
 #define _GNU_SOURCE
@@ -71,7 +71,7 @@
 #include <features.h>
 #include <stdio.h>
 #include <stdlib.h>
-#ifndef min
-#define min(a,b) (((a)<(b))?(a):(b))
-#endif
+// #ifndef min
+// #define min(a,b) (((a)<(b))?(a):(b))
+// #endif
 #endif /* Pegasus_Platform_LINUX_IX86_GNU_h */
